test/extended/operators/images.go

```diff
@@ -117,12 +117,18 @@ var _ = Describe("[sig-arch] Managed cluster", func() {
 					invalidPodContainerImagePullPolicy.Insert(fmt.Sprintf("%s/%s/%s imagePullPolicy=%s", pod.Namespace, pod.Name, container.Name, container.ImagePullPolicy))
 				}
 			}
+
+			if pod.Status.Phase != v1.PodRunning {
+				continue
+			}
 			// check if the container's image from the downstream.
 			for j := range pod.Spec.Containers {
 				containerName := pod.Spec.Containers[j].Name
 				commands := []string{
```

Comment:
I think the problem is that this checks the image is based on RHEL, but doesn't check that it comes from a trusted source. Do we want to hardcode the URL of the image registry in the tests or is there a better way?

Reply:
Thinking more about this, I think this should be enough to check that every image is pulled from the same registry and namespace as the Release Info image.